data/mainnet/pools.js

```diff
@@ -4,6 +4,181 @@ const addresses = require('./addresses.json')
 const strat15PercentFactor = '0.85'
 
 module.exports = [
+  {
+    chain: CHAIN_IDS.BASE,
```

Comment:
Can you add this new field for all pools you added?

`liquidityUrl: 'https://aerodrome.finance/connect?token0=0x2aF864fb54b55900Cd58d19c7102d9e4FA8D84a3&token1=0x4200000000000000000000000000000000000006&stable=false'`